api/client/node/ps.go

```diff
@@ -18,16 +18,20 @@ type psOptions struct {
 	filter    opts.FilterOpt
 }
 
-func newPSCommand(dockerCli *client.DockerCli) *cobra.Command {
+func newPsCommand(dockerCli *client.DockerCli) *cobra.Command {
 	opts := psOptions{filter: opts.NewFilterOpt()}
 
 	cmd := &cobra.Command{
-		Use:   "ps [OPTIONS] self|NODE",
-		Short: "List tasks running on a node",
-		Args:  cli.ExactArgs(1),
+		Use:   "ps [OPTIONS] [NODE]",
+		Short: "List tasks running on a node, default to self",
+		Args:  cli.RequiresRangeArgs(0, 1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			opts.nodeID = args[0]
-			return runPS(dockerCli, opts)
+			if len(args) == 0 {
```

Comment:
Actually, I misunderstood @crosbymichael. I prefer 
```
opts.NodeID = args[0]
if len(args) == 0 {
    opts.NodeID = "self"
}
```
As we already use `Args:  cli.RequiresRangeArgs(0, 1)`, I think it's no need to use a switch. WDYT? @stevvooe